### ISSUE TITLE
[hostagent] Waiting for the essential requirement 1 of 5: "ssh" on M1 Mac monterey

### ISSUE THREAD

**Issue by rjra2611** [abiosoft/colima#705]
### Description

Hey, 

I'm trying to run `colima start --cpu 4 --memory 8` after a fresh brew install on my M1 Mac monterey but it seems to hang

[serial (9).log](https://github.com/abiosoft/colima/files/11377525/serial.9.log)

Also tried `colima start --cpu 4 --memory 8 --cpu-type qemu64` which gave `error at 'creating and starting': exit status 1`

![image](https://user-images.githubusercontent.com/28739120/235778243-dba2709f-9e47-4cbf-9047-97ee6eeede2d.png)

Logs for `colima start --cpu 4 --memory 8 --arch x86_64`

[serial (10).log](https://github.com/abiosoft/colima/files/11377556/serial.10.log)

### Version

Colima Version:
- `colima version 0.5.4`

Lima Version:

![image](https://user-images.githubusercontent.com/28739120/235779757-b971810d-d25c-462a-87fa-d28e98cdc198.png)

Qemu Version:

- `qemu-img version 8.0.0`

![image](https://user-images.githubusercontent.com/28739120/235779319-ecb0ec3d-54bc-4abe-b2c6-6c98e3fe25c5.png)

Which colima

![image](https://user-images.githubusercontent.com/28739120/235777529-23fc5318-94de-4bd6-bf07-99f66590b780.png)

Which qemu-img

![image](https://user-images.githubusercontent.com/28739120/235777563-97379249-b728-4b0b-a219-a5549b64f844.png)

### Operating System

- [ ] macOS Intel <= 12 (Monterrey)
- [ ] macOS Intel >= 13 (Ventura)
- [X] macOS M1 <= 12 (Monterrey)
- [ ] macOS M1 >= 13 (Ventura)
- [ ] Linux

### Output of `colima status`

FATA[0000] colima is not running

### Reproduction Steps

1. brew install colima
2. colima start --cpu 4 --memory 8

### Expected behaviour

`colima start` should not hang

**Comment by abiosoft** [comment]
Can you share the output of `~/.lima/colima/ha.stderr.log` ?

**Comment by rjra2611** [comment]
Sure, there you go, I believe this is from the latest run which was using the command `colima start test --verbose`

[ha.stderr.log](https://github.com/abiosoft/colima/files/11400112/ha.stderr.log)

**Comment by abiosoft** [comment]
Looking at the logs, I found this which seems like the qemu process is getting terminated abruptly somehow.

```
{"level":"debug","msg":"qemu[stderr]: qemu-system-aarch64: terminating on signal 2 from pid 2301 (\u003cunknown process\u003e)","time":"2023-05-02T21:13:50+01:00"}
{"level":"info","msg":"Received SIGINT, shutting down the host agent","time":"2023-05-02T21:13:50+01:00"}
```

**Comment by abiosoft** [comment]
It's a bit strange that the `qemu-system-aarch64` binary is being used since you are using the `--arch x86_64`.

**Comment by rjra2611** [comment]
> Looking at the logs, I found this which seems like the qemu process is getting terminated abruptly somehow.

Anything I can do to get better logs to identify the reason? Could it be the log of the keyboard interrupt to stop the process as it was waiting for too long 

> It's a bit strange that the qemu-system-aarch64 binary is being used since you are using the --arch x86_64.

As I mentioned above these logs seems to be from the latest run where I tried `colima start test --verbose`

[ha.stderr.log](https://github.com/abiosoft/colima/files/11400112/ha.stderr.log)

**Comment by abiosoft** [comment]
@rjra2611 just to confirm, this only happens when you're simulating x86_64 right?

**Comment by rjra2611** [comment]
> this only happens when you're simulating x86_64 right?

Nope, as shared in the issue description. I tried the following:

1. `colima start --cpu 4 --memory 8`
2. `colima start --cpu 4 --memory 8 --arch x86_64`
3. `colima start test --verbose`

But got `Waiting for the essential requirement 1 of 5: "ssh"` for all of the above.

The only command which gave me a different result was `colima start --cpu 4 --memory 8 --cpu-type qemu64` as shared in the issue description 

![image](https://user-images.githubusercontent.com/28739120/236312773-1ec8b9f1-4cf0-4746-bc08-57074d4bf750.png)

**Comment by abiosoft** [comment]
You cannot change an architecture of an existing VM. If you initially created with `--arch x86_64` and stopped the VM, the next start would not change the architecture even if you specify a `--arch aarch64`. 

You will need to either create a new instance by specifying the profile name `colima start profile2` or you delete the existing profile `colima delete`.

**Comment by rfay** [comment]
@rjra2611 you should probably also make sure you've done these when testing:

* a successful `brew upgrade`
* `rm -r ~/.colima ~/.lima` 

I use colima 0.5.4 with limactl 0.15.1 every day on mac M1, so so do dozens or hundreds of other DDEV users, so there must be something odd going on. I don't have a place to test with Monterrey, but lots of people are using it.

The [setup we recommend for DDEV](https://ddev.readthedocs.io/en/latest/users/install/docker-installation/#macos) is `colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1`

**Comment by rjra2611** [comment]
Thanks, @rfay 

Something worked, something didn't. I tried:

1. `brew update`, `brew upgrade`
2. colima uninstall/reinstall
3. rm -r ~/.colima ~/.lima
4. colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1

But it doesn't work and says `--vm-type` flag not found. The help doesn't show the option as well. Colima is 0.5.4

![image](https://user-images.githubusercontent.com/28739120/236336353-54c75333-a7d9-4d98-8cc4-0f0c365a1ab2.png)





What worked was using `--arch x86_64`

Logs from `colima start profile4 --cpu 4 --memory 8 --arch x86_64`

[serial.log](https://github.com/abiosoft/colima/files/11401816/serial.log)
[ha.stderr.log](https://github.com/abiosoft/colima/files/11401817/ha.stderr.log)



As may require is **NOT** to start VM in x86_64 I tried 

`colima start --cpu 4 --memory 6 --disk 100 --mount-type=sshfs --dns=1.1.1.1` -- whcih didnt' work. Below are the logs for same.

[ha.stderr.log](https://github.com/abiosoft/colima/files/11401855/ha.stderr.log)
[serial.log](https://github.com/abiosoft/colima/files/11401856/serial.log)


NOTE: Used this after `colima delete` to start from fresh



CC @abiosoft

**Comment by rfay** [comment]
I didn't remember, maybe `--vm-type` only works on current Ventura. 

But running with --arch x68_64 is something which would be very, very unusual. 

Can you please show the output of `which colima` and of `file $(which colima)` ?

**Comment by rjra2611** [comment]
Sure, here you go @rfay 

![image](https://user-images.githubusercontent.com/28739120/236343984-5994b6c8-7dcd-451a-b578-f0e0dfdc7698.png)

Also, output of `colima start profile4 --cpu 4 --memory 8 --arch x86_64` if it helps 

![image](https://user-images.githubusercontent.com/28739120/236344104-16ea2478-dd9b-45c0-8501-c79b11db85a6.png)

**Comment by rfay** [comment]
I'm glad you have the arm64 colima, that's good, and you have homebrew installed in the right place; lots of people have trouble when they move to mac M1 because they copy /usr/local/bin with amd64 homebrew over to the new mac, and it doesn't work at all.

Your situation makes no sense at all :( 

You really don't want x86_64 right? You want to be running arm64?  x86_64 will be nothing but trouble and poor performance for you unless there's some exotic reason you really have to run x86_64/amd64.

I know you've already done this, but `colima start p5` just fails right, and hangs? That should bring up an arm64 profile (assuming you haven't used p5 before)

**Comment by abiosoft** [comment]
> I didn't remember, maybe --vm-type only works on current Ventura.

Yeah, it only works on Ventura.

**Comment by rjra2611** [comment]
@rfay sure there you go 

![image](https://user-images.githubusercontent.com/28739120/236498605-6e69cf48-00ea-4e4d-8a94-0f2efaa22194.png)

Yup, nothing in /usr/local/Homebrew .

![image](https://user-images.githubusercontent.com/28739120/236498711-fd30c9f7-a3e6-4a11-8294-a61dc002dbd3.png)

> I know you've already done this, but colima start p5 just fails right, and hangs? That should bring up an arm64 profile (assuming you haven't used p5 before)

Yup, tried it as shared above with logs, sharing the screenshot here as well.

![image](https://user-images.githubusercontent.com/28739120/236499493-cc73da6c-e7dc-4dea-888c-9294ae36fcb2.png)


@abiosoft can you please look into it too. My use case is to only run colima with arm64 but currently only works when using `x86_64 ` which seems like a very popular solution as per the comment [here](https://github.com/abiosoft/colima/issues/424#issuecomment-1335912905)

**Comment by rfay** [comment]
That is totally bizarre. Thanks for trying it out.

**Comment by AlexeyRaga** [comment]
I have upgraded to `colima-0.5.4` and am having a similar issue as well.
I am on an Intel MacOS, have done `colima delete` and am having this:

```
$ colima start --very-verbose

TRAC[0000] cmd ["limactl" "info"]
TRAC[0000] cmd ["limactl" "list" "colima" "--json"]
TRAC[0000] error retrieving running instance: instance 'colima' does not exist
INFO[0000] starting colima
INFO[0000] runtime: docker
INFO[0000] preparing network ...                         context=vm
TRAC[0000] cmd ["/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/colima" "daemon" "status" "default"]
TRAC[0000] cmd ["/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/colima" "daemon" "start" "default" "--gvproxy" "--verbose"]
INFO[0000] creating and starting ...                     context=vm
TRAC[0000] cmd ["limactl" "start" "--tty=false" "/tmp/colima.yaml"]
> Terminal is not available, proceeding without opening an editor
> "Attempting to download the image from \"https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.0-2/alpine-lima-clm-3.16.2-x86_64.iso\"" digest="sha512:e9e118498f3a0745574ffc3686105d2c9777f7142164fe50ee47909dabd504c80ac29aeb76f9582706173310d1488d3b6f0ee9018e2a6aadc28eefb7767b63ec"
> Using cache "/Users/alexey/Library/Caches/lima/download/by-url-sha256/e1cbaef56fc8e75af106d934389a17618643fcecfa066302d45ffb0390709851/data"
> [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/alexey/.lima/colima/serial.log")
> SSH Local Port: 50229
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Driver stopped due to error: "exit status 255"
> [hostagent] Shutting down the host agent
> [hostagent] failed to exit SSH master
> [hostagent] Shutting down QEMU with ACPI
> [hostagent] failed to open the QMP socket "/Users/alexey/.lima/colima/qmp.sock", forcibly killing QEMU
> [hostagent] failed to kill QEMU
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
```

**Comment by aelsnz** [comment]
What might be good is cleanup and restart - and yes, restart sounds not needed, but if you had a system running for a while, and updated software etc - a clean restart might not be a bad idea.

What might be good to do:

1. Make sure no lima or colima VMs are running 
2. Cleanup `~/.colima` and `~/.lima` 
3. Cleanup the cache here - $HOME/Library/Caches/lima/download/
4. Make sure you run "brew update; brew upgrade" and that all is on latest versions.
6. Now close all terminals and apps and reboot.  
7. When reboot is done, and logged in give it a minute - let all the MacOs background processes start.

Now try a basic setup:  `colima start -c2 -d10 -m2 --very-verbose` and see what happens.

Also if any issues - maybe post the following so all is 100% sure what versions are in use:

```
sw_vers
uname -a
colima version
lima --version
qemu-img -V
/opt/colima/bin/socket_vmnet -version
```

**Comment by AlexeyRaga** [comment]
No, it did not work still :(

```
$ sw_vers
ProductName:		macOS
ProductVersion:		13.3.1
ProductVersionExtra:	(a)
BuildVersion:		22E772610a

$ uname -a
Darwin MacBook-Pro.local 22.4.0 Darwin Kernel Version 22.4.0: Mon Mar  6 21:00:17 PST 2023; root:xnu-8796.101.5~3/RELEASE_X86_64 x86_64 i386 Darwin

$ colima version
colima version 0.5.4
git commit: feef417

$ lima --version
limactl version 0.15.1

$ qemu-img -V
qemu-img version 8.0.0
Copyright (c) 2003-2022 Fabrice Bellard and the QEMU Project developers
```

And here are logs:

```
$ colima start -c2 -d10 -m2 --very-verbose
TRAC[0000] cmd ["limactl" "info"]
TRAC[0000] cmd ["limactl" "list" "colima" "--json"]
TRAC[0000] error retrieving running instance: instance 'colima' does not exist
INFO[0000] starting colima
INFO[0000] runtime: docker
INFO[0000] preparing network ...                         context=vm
TRAC[0000] cmd ["ln" "-sf" "/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/.colima-wrapped" "/Users/alexey/.colima/_wrapper/d7f6c041b026e94787f11d5084bf50491a1e6e02/bin/qemu-system-aarch64"]
TRAC[0000] cmd ["ln" "-sf" "/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/.colima-wrapped" "/Users/alexey/.colima/_wrapper/d7f6c041b026e94787f11d5084bf50491a1e6e02/bin/qemu-system-x86_64"]
TRAC[0000] cmd ["ln" "-sf" "/nix/store/mwl0pjmcddcm7wfplyi581q645xfp1iz-qemu-8.0.0/share" "/Users/alexey/.colima/_wrapper/d7f6c041b026e94787f11d5084bf50491a1e6e02/share"]
TRAC[0000] cmd ["/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/colima" "daemon" "status" "default"]
TRAC[0000] cmd ["/nix/store/gra06z3s0ysxl1zzx11b5d6zi2z23pg5-colima-0.5.4/bin/colima" "daemon" "start" "default" "--gvproxy" "--verbose"]
INFO[0000] creating and starting ...                     context=vm
TRAC[0000] cmd ["limactl" "start" "--tty=false" "/tmp/colima.yaml"]
> Terminal is not available, proceeding without opening an editor
> "Attempting to download the image from \"https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.0-2/alpine-lima-clm-3.16.2-x86_64.iso\"" digest="sha512:e9e118498f3a0745574ffc3686105d2c9777f7142164fe50ee47909dabd504c80ac29aeb76f9582706173310d1488d3b6f0ee9018e2a6aadc28eefb7767b63ec"
>
> 18.18 MiB / 290.00 MiB (6.27%) ? p/s
> 40.45 MiB / 290.00 MiB (13.95%) 4.45 MiB/s
> 62.61 MiB / 290.00 MiB (21.59%) 4.45 MiB/s
> 86.37 MiB / 290.00 MiB (29.78%) 4.47 MiB/s
> 108.07 MiB / 290.00 MiB (37.27%) 4.46 MiB/s
> 131.12 MiB / 290.00 MiB (45.21%) 4.47 MiB/s
> 151.78 MiB / 290.00 MiB (52.34%) 4.45 MiB/s
> 171.09 MiB / 290.00 MiB (59.00%) 4.41 MiB/s
> 194.01 MiB / 290.00 MiB (66.90%) 4.42 MiB/s
> 219.23 MiB / 290.00 MiB (75.60%) 4.46 MiB/s
> 243.12 MiB / 290.00 MiB (83.84%) 4.48 MiB/s
> 266.40 MiB / 290.00 MiB (91.86%) 4.50 MiB/s
> 287.43 MiB / 290.00 MiB (99.12%) 4.48 MiB/s
> 290.00 MiB / 290.00 MiB (100.00%) 4.79 MiB/stime="2023-05-08T17:03:05+10:00" level=info msg="Downloaded the image from \"https://github.com/abiosoft/alpine-lima/releases/download/colima-v0.5.0-2/alpine-lima-clm-3.16.2-x86_64.iso\""
> [hostagent] Starting QEMU (hint: to watch the boot progress, see "/Users/alexey/.lima/colima/serial.log")
> SSH Local Port: 56136
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Driver stopped due to error: "exit status 255"
> [hostagent] Shutting down the host agent
> [hostagent] failed to exit SSH master
> [hostagent] Shutting down QEMU with ACPI
> [hostagent] failed to open the QMP socket "/Users/alexey/.lima/colima/qmp.sock", forcibly killing QEMU
> [hostagent] failed to kill QEMU
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
```

**Comment by rfay** [comment]
It does look like you have qemu running already in this situation. We have asked about limactl current version a number of times, but not about qemu. You might want to brew uninstall all your qemu dependencies and reinstall them, and make sure qemu isn't running at all.

But these lines sure do seem to imply problems outside of colima:

```
> [hostagent] Driver stopped due to error: "exit status 255"
> [hostagent] Shutting down the host agent
> [hostagent] failed to exit SSH master
> [hostagent] Shutting down QEMU with ACPI
> [hostagent] failed to open the QMP socket "/Users/alexey/.lima/colima/qmp.sock", forcibly killing QEMU
> [hostagent] failed to kill QEMU
```

**Comment by rjra2611** [comment]
> My use case is to only run colima with arm64 but currently only works when using x86_64  which seems like a very popular solution as per the comment https://github.com/abiosoft/colima/issues/424#issuecomment-1335912905


@rfay As stated earlier that I'm only able to run Colima in `x86_64` mode but my aim is to use an Amd64 image using the Colima VM. Would you recommend going ahead with this setup -> Colima `x86_64` mode on Apple Silicom M1/M2 (arm64) trying to run an Amd64 image?

**Comment by rfay** [comment]
Hi @rjra2611 - All I can say is that your mileage may vary. In general, performance with emulated containers is awful and in many cases breaks unpredictably. For example, people are always trying to run Oracle DB and Oracle MySQL containers (only available in amd64) on arm64 and they run into complete blockers or random failures. 

I would certainly recommend that you upgrade to macOS Ventura but I assume there's a reason you haven't done that. It would sure help to understand this issue. But my expectation is that there's something in your network or computer that is the problem, and it's probably not Monterrey.

But have you already tried temporarily disabling all firewall/VPN software? Have you tried using your computer on a different network?

**Comment by rjra2611** [comment]
@rfay Thank you, that's very helpful

**Comment by whaler-ragweed** [comment]
I faced this issue every time I set `--memory x`.

**Comment by reesericci** [comment]
I have this exact same issue, but on an _intel_ 2020 macbook air.

**Comment by chrisRedwine** [comment]
Similar to the last two comments, I also run into this issue on an Intel MacBook Pro running 13.4 with 64gb RAM, but only whenever I run with `--memory x` where `x` is close to the free memory available (for me at the time, `--memory 30` works but `--memory 31` doesn't).

**Comment by ehakan** [comment]
Same issue here

Tried completely cleaning colima, restarting the machine. generally using the options:

```
colima start --arch aarch64 --vm-type=vz --vz-rosetta --cpu 4
```

Modifying memory doesn't change anything for me, it still waits on "ssh" and fails with the same logs

**Comment by AndrewIISM** [comment]
I have same issue with same issue with Apple M2 Max. Version of MacOs is `13.5.1`.

```
colima start --cpu 6 --memory 8 --disk 100 --arch x86_64        
INFO[0000] starting colima                              
INFO[0000] runtime: docker                              
INFO[0000] preparing network ...                         context=vm
INFO[0000] creating and starting ...                     context=vm
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> [hostagent] Waiting for the essential requirement 1 of 5: "ssh"
> did not receive an event with the "running" status
FATA[0600] error starting vm: error at 'creating and starting': exit status 1
```

**Comment by masrawi** [comment]
same issue with Apple M2 Max MacOs 13.5.1 (22G90)

**Comment by AndrewIISM** [comment]
I fixed it with downgrade qemu to to v8.0.3
https://github.com/abiosoft/colima/issues/786#issuecomment-1695281548

**Comment by tdy** [comment]
> I fixed it with downgrade qemu to v8.0.3 [#786 (comment)](https://github.com/abiosoft/colima/issues/786#issuecomment-1695281548)

That looks like a different error ("failed to open QMP socket") and different chip (Intel).

Tried downgrading qemu anyway, but it didn't fix this error ("waiting for essential requirement ssh") on my system (M2 Ventura).

**Comment by MattMacGregor** [comment]
> I fixed it with downgrade qemu to v8.0.3 [#786 (comment)](https://github.com/abiosoft/colima/issues/786#issuecomment-1695281548)

Using a M2 Max on Ventura 13.2.1, this method worked for me (used Option 1 from the linked comment). Figured it was worth mentioning

**Comment by StephanMeijer** [comment]
Also encountered this issue. Removed `~/.lima` and `/.colima` but didn't fix my issues.

```
$ sw_vers
ProductName:		macOS
ProductVersion:		13.4.1
BuildVersion:		22F82

$ uname -a
Darwin Stephans-MacBook-Pro.local 22.5.0 Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:20 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6000 arm64

$ colima version
colima version v0.5.6
git commit: 

$ lima --version
FATA[0000] unknown flag: --version                      

$ qemu-img -V
qemu-img version 8.0.5
Copyright (c) 2003-2022 Fabrice Bellard and the QEMU Project developers

$ /opt/colima/bin/socket_vmnet -version
versions.sh: line 19: /opt/colima/bin/socket_vmnet: No such file or directory
```

As I am using MacPorts, I have no possibility downgrading to `8.0.3`.

**Comment by M4urici0GM** [comment]
Same issue here. 😢 
tried downgrading to 8.0.3 but without success.

**Comment by rfay** [comment]
Now for the first time ever I've seen this, was wondering if it had to do with QEMU update, https://github.com/Homebrew/homebrew-core/pull/155388

**Comment by rfay** [comment]
This latest seems to be due to QEMU homebrew upgrade etc, 
* https://gitlab.com/qemu-project/qemu/-/issues/1990#note_1666099003
* https://github.com/Homebrew/homebrew-core/pull/155388

**Comment by M4urici0GM** [comment]
Same issue on Macos Sonoma 14.4.1 on M3 Pro 
Only docker desktop works, but uses too much ram out of the box D:

**Comment by M4urici0GM** [comment]
Apparently, the workaround is using vz instead of qemu for now, on lima's issue [#1996](https://github.com/lima-vm/lima/issues/1996) 
It worked for me
```colima start --vm-type vz```

**Comment by royalan** [comment]
Try removing the cache folder `$HOME/Library/Caches/lima`. I fixed the problem by doing this.

**Comment by szhem** [comment]
I've had the same issue and looking through the /Users/<user>/.colima/_lima/colima/ha.stderr.log found that /Users/<user>/.colima/_lima/_config/user ssh key is too open, e.g. has to wide permissions.

After executing
```bash
chmod o-r,g-r /Users/szhemzhitsky/.colima/_lima/_config/user
```
everything was able to start without issues

**Comment by elisagia** [comment]
> try removing the existing instance `colima delete` then start with vz `colima start --vm-type vz`

this worked for me! thanks!

**Comment by dankeller** [comment]
> I've had the same issue and looking through the /Users/<user>/.colima/_lima/colima/ha.stderr.log found that /Users/<user>/.colima/_lima/_config/user ssh key is too open, e.g. has to wide permissions.
> 
> After executing
> 
> ```shell
> chmod o-r,g-r /Users/<user>/.colima/_lima/_config/user
> ```
> 
> everything was able to start without issues

Trying the other items in this thread and nothing worked until I saw this. The `user` key was globally readable. I ran `chmod o-r .colima/_lima/_config/user` during the startup process and that unstuck it.

```
~ ❯❯❯ ls -la .colima/_lima/_config/user                                                                                                                       ⏎
-rw----r--  1 user  group  411 Mar 22 16:33 .colima/_lima/_config/user
~ ❯❯❯ chmod o-r .colima/_lima/_config/user
```

The "waiting for ssh" message in my other terminal tab went away and colima finished starting!

I'm on MacOs Sonoma | M1
```
colima version 0.6.8
git commit: 9b0809d0ed9ad3ff1e57c405f27324e6298ca04f

runtime: docker
arch: aarch64
client: v26.1.1
server: v24.0.9
```

**Comment by amadaCriollo** [comment]
> Tuve el mismo problema y al revisar /Users/<user>/.colima/_lima/colima/ha.stderr.log encontré que la clave ssh de /Users/<user>/.colima/_lima/_config/user está demasiado abierta, por ejemplo, tiene permisos demasiado amplios.
> 
> Después de ejecutar
> 
> ```shell
> chmod o-r,g-r /Users/<user>/.colima/_lima/_config/user
> ```
> 
> Todo pudo comenzar sin problemas.

Solvento el problema, muchas gracias
